docs/source/modules/tcms.xmlrpc.api.rst

```diff
@@ -31,9 +31,9 @@ Submodules
    tcms.xmlrpc.api.product
    tcms.xmlrpc.api.tag
    tcms.xmlrpc.api.testcase
-   tcms.xmlrpc.api.testcaserun
-   tcms.xmlrpc.api.testcaserunstatus
```

Comment:
How about removing the sources for modules with the old names (or actually renaming them instead of adding new ones to keep git history).

And changing the commit log to something more saner.